### PR TITLE
Show node address type in kubectl describe node command

### PR DESCRIPTION
**Pull request by xiangpengzhao**
**What this PR does / why we need it**:
To make node addresses more clear when describing node.

Before:
```
Addresses:		192.168.1.220,192.168.1.220,192.168.1.220
```

After:
```
Addresses:
 LegacyHostIP:	192.168.1.220
 InternalIP:	192.168.1.220
 Hostname:	192.168.1.220
```

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: pkg/printers/internalversion/describe.go [hunk 1]
```diff
@@ -2048,9 +2048,10 @@ func describeNode(node *api.Node, nodeNonTerminatedPodsList *api.PodList, events
 					c.Message)
 			}
 		}
-		addresses := make([]string, 0, len(node.Status.Addresses))
+
+		w.Write(LEVEL_0, "Addresses:\n")
 		for _, address := range node.Status.Addresses {
-			addresses = append(addresses, address.Address)
+			w.Write(LEVEL_0, " %s:\t%s\n", address.Type, address.Address)
```
Comment (fabianofranz) [inline pkg/printers/internalversion/describe.go]: Use `LEVEL_1` which will indent it automatically, and remove your own space at the beginning of the string.

**Comment by xiangpengzhao** [comment]
Inline comment addressed. Thanks @fabianofranz !

**Comment by fabianofranz** [comment]
@k8s-bot ok to test

**Comment by fabianofranz** [comment]
@xiangpengzhao thanks for the pr!
/lgtm
/approve

**Pull request merged**